app/common/commonMenu.js

```diff
@@ -143,8 +143,10 @@ module.exports.printMenuItem = () => {
 }
 
 module.exports.simpleShareActiveTabMenuItem = (l10nId, type, accelerator) => {
+  const siteName = type.charAt(0).toUpperCase() + type.slice(1)
+  
   return {
-    label: locale.translation(l10nId),
+    label: locale.translation(l10nId).replace(/{{\s*siteName\s*}}/, siteName),
```

Comment:
Can you change this one to `locale.translation(l10nId, {siteName: siteName})`? We already have a mechanism for variables in the translations.

Reply:
Yes

Reply:
I tried your suggestion when I lookup locale.translation it only accepted one argument.

Reply:
not sure that I follow, this change is not working for you?

Reply:
The change is not working for me.

Reply:
ok, let me try it

Reply:
ok I see what's going on, will fix it, so that this will work as well

Reply:
@sergiorojasa ok this should be working now, can you please check out if everything is ok now? I pushed my fix commit to your PR